src/Files.App/Views/PaneHolderPage.xaml.cs

```diff
@@ -192,6 +191,9 @@ public bool IsCurrentInstance
 		public PaneHolderPage()
 		{
 			InitializeComponent();
+
+			UserSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
```

Comment:
Can you inject the service in line 191?

Reply:
oh that’s not good idea I suppose. Always have to initialize Xaml contents first.

I’d like to hear some opinion about it. @hez2010

Reply:
We want to inject the service like we've been doing in other areas of the app `public PaneHolderPages(service...)`